python/list-streams/README.md

```diff
@@ -0,0 +1,72 @@
+# Kinesis Video Streams Counter Lambda Function
+
+This AWS Lambda function Demonstrates how to find the total number of KVS stream resources in any AWS Account using the [`ListStreams`](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_ListStreams.html) API. This helps in monitoring the total streams usage to request limit increases as needed.
+
+## Functionality
+
+The Lambda function performs the following tasks:
+
+1. Connects to the Kinesis Video Streams service in the specified AWS region.
+2. Retrieves the list of Kinesis Video Streams using the [`ListStreams`](https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_ListStreams.html) API.
+3. Iterates through the paginated responses to count the total number of streams.
+4. Provides retry logic for transient errors such as throttling or service limits.
+5. Returns the total count of Kinesis Video Streams.
+
+Note: the total count is retrievable within a single Lambda run.
```

Comment:
/nit Assumption:  The total number of streams should be retrievable by a single Lambda execution.